operating_unit/models/operating_unit.py

```diff
@@ -51,6 +51,8 @@ def name_search(self, name="", args=None, operator="ilike", limit=100):
         names2 = []
         if name:
             domain = [("code", "=ilike", name + "%")]
+            if args:
+                domain.extend(list(args))
```

Comment:
Would be possible a case when args can be not transformed into a list?

Reply:
I don't think so, even in default Odoo implementation of `name_search` `args` are treated as list or `None` only

Reply:
I see. Can you please change the commit message so it begins with `[FIX] operating_unit: ` ? then I think it is fine

Reply:
Amended commit message

Reply:
Would be nice to have this fix in branches 15.0, 16.0 and 13.0